### DISCUSSION TITLE
Aryan

### DISCUSSION THREAD

**Discussion by sanjay-kv** [recodehive/Stackoverflow-Analysis#366] [General]
s

**Comment by sanjay-kv** [comment]
[aryan.pdf](https://github.com/user-attachments/files/17005867/aryan.pdf)
[ATS Version of CV (14).docx](https://github.com/user-attachments/files/17005868/ATS.Version.of.CV.14.docx)

**Comment by Aryan9901** [comment]
helo sir, thank you for reviewing my resume i have updated my resume

https://drive.google.com/file/d/1jrcTBnDz71aVy9ceFWoT3MWtAbTJ-cK6/view?usp=drivesdk

i am still confused how to mentioned the soft skills please check mentioned soft skills in resume.

my portfolio link is
https://aryang.xyz/

And also sir what tool you uses to score ats of resume I am currently using resume worded

**Reply by sanjay-kv** [reply]
i too use resume worded, i use some internal tools from my workplace to do the same.

**Reply by sanjay-kv** [reply]
Soft skills : should be keywords like Team management, Collaborative, Inclusive, Public speaking, design etc. 

These skills should change according to job description.

**Reply by Aryan9901** [reply]
Sir once review this resume I have updated it

**Reply by sanjay-kv** [reply]
the current one is good.
